alerts/openshift-virtualization-operator/VirtApiRESTErrorsHigh.md

```diff
@@ -36,7 +36,7 @@ affected.
 4. Obtain the details of the `virt-api` pods:
 
    ```bash
-   $ oc describe -n $NAMESPACE <virt-api>
+   $ oc describe pod -n $NAMESPACE <virt-api>
```

Comment:
use pod selector here and in step 3

Reply:
see previous comment at `alerts/openshift-virtualization-operator/VirtApiRESTErrorsBurst.md`